SR2020/2020-03-05-coronavirus.md

```diff
@@ -0,0 +1,21 @@
+---
+to: sr2020-teams
+subject: Update on Coronavirus and SR
+---
+
+# Update on SR and the COVID-19 Coronavirus
+
+Hello everyone, 
+
+TL;DR: We're not intending to cancel the event, but will be monitoring the situation closely and will be taking action both during and before the event to minimize risk.
+
+I'm sure you've heard about the threat caused by the COVID-19 coronavirus. and you've likely heard of many large events being cancelled due to concerns of spreading of the virus.
+
+At the current time we're *not* intending to cancel Student Robotics 2020 competition. However, we're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. We are also working on a back-up plan should the event be unable to run. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
+
+With an abundance of caution, we've decided to take action during the competition to minimise risk. We will frequently disinfect any touch points (i.e. door handles & surfaces) during the competition, and will try to source hand sanitizer for use during the event, (however there is currently a global shortage). We'll also be briefing all volunteers and attendees about best hygene practises. We'd also like to request every attendee who has recently visited or will arrive from another country to follow the 'returning travellers' section of the [UK Government guidance][returning-travellers].
```

Comment:
The advice currently provided by the government says nothing about regularly santising surfaces, and recommends soap and water with hand sanitiser as a fall back. Soap and water should be readily available in the toilet facilities. Do not forget that, as a public venue, Reading Uni will have to be taking relevant precautions for all events at their venue anyway!

Additionally, and more crucially, the competition is still 6 weeks away, the advice could change hugely in that time. I **strongly** recommend not committing us to any specific actions. Just say that we will keep ourselves informed of the current situation, and take any appropriate actions.

Reply:
I think it's fine to go more general as Scarzy says. The main point is for teams to know we are actively thinking about it.

Reply:
Maybe just something like "We're following the recommended advise" is enough, rather than specifics.

Reply:
'following recommended advice' is the bare minimum here. We should do more to ensure the event is safe and to settle the worries of the teams.

I've left in the above section but have clarified that it may change (therefore not committing us to any of it)

Reply:
I'm outvoted, so I'll remove it